src/main/java/com/bravo/user/dao/model/User.java

```diff
@@ -33,6 +40,10 @@
   @Column(name = "updated", nullable = false)
   private LocalDateTime updated;
 
+  @OneToOne(cascade = CascadeType.REMOVE, fetch = FetchType.LAZY)
```

Comment:
Bonus points for adding the CascadeType.  Would like to know why this is lazily fetched.

Reply:
Lazy fetch due to discussion in email - agreed that the API for loading users would not be loading the profile entity as part of it. If API would need to load it together in the future, the query would need to call out the desire to fetch that profile.